src/ur5e_arm.hpp

```diff
@@ -113,47 +82,29 @@ class UR5eArm final : public Arm, public Reconfigurable {
     std::vector<GeometryConfig> get_geometries(const ProtoStruct&) override {
         throw std::runtime_error("unimplemented");
     }
+
+   private:
+    struct state_;
+
     enum class UrDriverStatus : int8_t  // Only available on 3.10/5.4
     {
         NORMAL = 1,
         ESTOPPED = 2,
         READ_FAILURE = 3,
         DASHBOARD_FAILURE = 4
     };
+
```

Comment:
It may be possible to forward declare UrDriverStatus, but I'm not 100%. Let's try it together and see.

Reply:
was able to move to definition to cpp